admin/securedrop_admin/__init__.py

```diff
@@ -544,10 +548,68 @@ def sdconfig(args):
     return 0
 
 
+def generate_new_v3_keys():
+    """This function generate new keys and returns them as tuple.
+
+    :returns: Tuple(public_key, private_key)
+    """
+
+    private_key = x25519.X25519PrivateKey.generate()
+    private_bytes = private_key.private_bytes(
+        encoding=serialization.Encoding.Raw	,
+        format=serialization.PrivateFormat.Raw,
+        encryption_algorithm=serialization.NoEncryption())
+    public_key = private_key.public_key()
+    public_bytes = public_key.public_bytes(
+        encoding=serialization.Encoding.Raw,
+        format=serialization.PublicFormat.Raw)
+
+    public = base64.b32encode(public_bytes)[:-4].decode("utf-8")
+    private = base64.b32encode(private_bytes)[:-4].decode("utf-8")
+    return public, private
+
+
+def get_v3_keys(filepath):
+    """
+    Returns the stored v3 public and private keys as Tuple.
+
+    :returns: Tuple(public_key, private_key)
+    """
+    with open(filepath) as fobj:
+        data = json.load(fobj)
+
+    return data
```

Comment:
Nit: don't use `data` as a variable name—all variables contain data! :slightly_smiling_face:

Reply:
I will update.

Reply:
resolved in `v3 onions: remove magic number for removing base32 padding characters`